ui/app/src/components/DialogTitle.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import makeStyles from '@material-ui/styles/makeStyles';
+import { Theme } from '@material-ui/core';
+import { palette } from '../styles/theme';
+import MuiDialogTitle from '@material-ui/core/DialogTitle/DialogTitle';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/CloseRounded';
+import React from 'react';
+
+const dialogTitleStyles = makeStyles((theme: Theme) => ({
+  titleRoot: {
+    margin: 0,
+    padding: '13px 20px 11px',
+    background: palette.white
+  },
+  title: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center'
+  },
+  subtitle: {
+    fontSize: '14px',
+    lineHeight: '18px',
+    paddingRight: '35px'
+  },
+  closeIcon: {}
+}));
+
+export interface DialogTitleProps {
+  title: string;
+  subtitle?: string;
+  icon?: any;
+  onClose(): void;
+}
+
+export default function DialogTitle(props: DialogTitleProps) {
+  const classes = dialogTitleStyles({});
+  const { onClose, title, subtitle, icon: Icon = CloseIcon } = props;
+  return (
+    <MuiDialogTitle disableTypography className={classes.titleRoot}>
+      <div className={classes.title}>
+        <Typography variant="h6">{title}</Typography>
```

Comment:
add props (with default values) for titleTypographyProps, subtitleTypographyProps. Put variants in there